app/gui/config/src/lib.rs

```diff
@@ -60,5 +60,6 @@ ensogl::read_args! {
         authentication_enabled : bool,
         email                  : String,
         application_config_url : String,
+        workflow               : String,
```

Comment:
The rest of the entires in this list is understandable by name, `workflow` is not (at least to me). Let's add here a short doc explaining what it is for. Maybe we should also rename it to `test_workflow` or `test_scenario`, what do you think?

Reply:
Renamed it to `test_workflow`. Extended the macro to support doc comments.